spec/spec_helper.rb

```diff
@@ -8,7 +8,7 @@ def compare(input, expected)
   # TODO:  Figure out a better way to do html compare in ruby..
   # this is overly dependent on things like class ordering, making it
   # fragile
-  output_str = REXML::Document.new(output).to_s.gsub(/\s/, '')
-  expected_str = REXML::Document.new(expected).to_s.gsub(/\s/, '')
+  output_str = Nokogiri::XML(output).to_s.gsub(/\s/, '')
+  expected_str = Nokogiri::XML(expected).to_s.gsub(/\s/, '')
```

Comment:
@kball found this thread which is related: http://stackoverflow.com/questions/1430102/how-do-i-test-xml-equality-in-ruby

Looks like you have to add a library... [nokogiri-diff](https://github.com/postmodern/nokogiri-diff) perhaps?